LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2012 Maxime Bouroumeau-Fuseau
+Copyright (C) 2012-2017 Maxime Bouroumeau-Fuseau, and ORCAS
```

Comment:
@brad from what I found on stackoverflow, we want to say
```
Original work Copyright (C) 2012 Maxime Bouroumeau-Fuseau
Modified work Copyright (C) 2017 ORCAS
```

https://softwareengineering.stackexchange.com/questions/157968/how-to-manage-a-copyright-notice-in-an-open-source-project